internal/setup/cache_options.go

```diff
@@ -69,31 +94,32 @@ func (c *KcpCacheOptions) GetCacheOptions() cache.Options {
 					c.kcpNamespace: {},
 				},
 			},
-			&certmanagerv1.Issuer{}: {
-				Namespaces: map[string]cache.Config{
-					c.kcpNamespace:   {},
-					c.istioNamespace: {},
-				},
-			},
```

Comment:
Issuer has been removed entirely since it is not needed anylonger. The Certificate value is provided depending on the cert-management solution in use. See getCertManagementCacheObjects